ui/components/app/nfts-items/nfts-items.js

```diff
@@ -169,12 +169,14 @@ export default function NftsItems({
         event: MetaMetricsEventName.sendAssetSelected,
         category: MetaMetricsEventCategory.Send,
         properties: {
-          ...sendAnalytics,
           is_destination_asset_picker_modal: false,
           new_asset_symbol: nft.name,
           new_asset_address: nft.address,
```

Comment:
Should we move these as well? NFT address at least is an on-chain reference.

Reply:
I might also be preferable to keep this data together in our metrics system but I'm not sure, depends on how its used.

Reply:
We can move it, nfts can be pretty identifying at times if they're lower volume